lom/src/plugins/plugins_common/plugins_common.go

```diff
@@ -161,7 +163,7 @@ func (gpl *PluginMetadata) CheckMisbehavingPlugins(pluginKey string) bool {
     }
 
     // Remove expired responses from the window(slice)
-    threshold := now.Add(-MAX_PLUGIN_RESPONSES_WINDOW_TIMEOUT_DEFAULT) // go back in time by duration
+    threshold := now.Add(-gpl.MaxPluginResponsesWindowTime) // go back in time by duration
     for i := 0; i < len(responses); i++ {
         if responses[i].Before(threshold) {
             responses = responses[i+1:]
```

Comment:
Just count the number to skip only
do just one assignment outside the loop. If any copy/realloc may happen - cut to once

responses = responses[skip:]

Reply:
responses = responses[i+1:]   in golang this is just change in pointer reference right ? there is no copy here ?

Reply:
TODO